lib/foundation/README.md

```diff
@@ -0,0 +1,5 @@
+# About This Build of Foundation
+
+*Version*: 6.3.1
```

Comment:
It would be nice to include more details in this `README.md` such as the link to the foundation custom download site and even steps. The goal is that future maintainers can follow your steps to easily upgrade Foundation.